.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: CI
```

Comment:
No more separate workflows for build-main and test-and-lint

This does both! And handles publishing